README.md

```diff
@@ -1187,7 +1187,7 @@ List of samples:
 - Update variable which defines the path to the load file/opticon file (e.g. $opticonFilePath) according to the location where you copied sample data.
 
         $workspaceId = 1000000
-        $loadFilePath = "\\files\T001\Files\SampleDataSources\load_file_01.dat"
+        $loadFilePath = "\\\\files\T001\StructuredData\\Import\SampleDataSources\load_file_01.dat"
```

Comment:
```suggestion
        $loadFilePath = "\\\\files\\T001\\StructuredData\\Import\\SampleDataSources\\load_file_01.dat"
```

Reply:
For rest samples there should be only one slash - I didn't notice incorrect, additional slash - thanks for noticing.